_posts/2021-07-02-newsletter-13.markdown

```diff
@@ -0,0 +1,37 @@
+
+### The Persistent Pre-Marathon Dream
+
+Every month or so I have a recurring dream that involves running a marathon. In these dreams, there is always some obstacle that either keeps me from getting to the start on time or prevents me from making progress effectively once the race has kicked off. For example, I had a dream the other night where I began a marathon with a course that took runners immediately inside of a sporting goods store and through a backroom warehouse before exiting to the outdoor world again. For some reason, I was late getting to the start of the race, and by the time I arrived, most of the runners were far ahead. The organizers had already begun closing down that part of the course and locking up the doors to the warehouse, which I still needed to pass through to complete the race. I started yelling at the organizers for designing such a terrible course. In the dream, I imagined that there had been a huge bottleneck trying to make it through this tiny warehouse opening, and the fact I saw no one else alongside me by the doors only intensified my realization that I was _very_ behind in the race.
+
+I've had other similar dreams that don't add up to anything either. In some cases, I am on the course, and I somehow lose my way and am not able to make it back on track. In others, I wake up and head to the start, only to realize after getting there that I had not in fact actually registered for the race. A sidebar: I did actually make this mistake in real life once, but not for a marathon, thankfully.
+
+Why do I share these things? I'm mainly curious if other runners also experience these nonsensical dreams about running. And because I think it's funny. It's a slight diversion from what has otherwise been a tough month or so of running news. We've already seen Shelby Houlihan get disqualified from two Olympics cycles because she tested positive for a banned anabolic steroid, which she claims is [attributable to a burrito she ate](https://www.nytimes.com/2021/06/15/sports/olympics/shelby-houlihan-burrito-olympics.html) back in December, 2020. Then, this week, U.S. sprinter Sha'Carri Richardson [tested positive for marijuana](https://www.nytimes.com/2021/07/02/sports/olympics/shacarri-richardson-marijuana-doping.html), which is somehow classified as a performance enhancing drug. The violation may also cost her a spot in the Olympics this summer. Both of these stories are frustrating, for different reasons to me. I don't think that Richardson should face a penalty for smoking marijuana.
```

Comment:
Ooh. Good. I had a quick note on Richardson in my draft. I'll take it out. You could add that it was legal to smoke marijuana recreationally in the state where she tested positive and that her mother had just died.